.gitignore

```diff
@@ -1,11 +1,7 @@
 *.pyc
-_trial_temp
```

Comment:
OK, it's not that big a file, and I don't think it helped that much, but re-ordered.